src/Symfony/Component/HttpClient/CachingHttpClient.php

```diff
@@ -49,13 +49,9 @@ public function __construct(HttpClientInterface $client, StoreInterface $store,
         $kernel = new HttpClientKernel($client);
         $this->cache = new HttpCache($kernel, $store, null, $defaultOptions);
 
-        unset($defaultOptions['debug']);
-        unset($defaultOptions['default_ttl']);
-        unset($defaultOptions['private_headers']);
-        unset($defaultOptions['allow_reload']);
-        unset($defaultOptions['allow_revalidate']);
-        unset($defaultOptions['stale_while_revalidate']);
-        unset($defaultOptions['stale_if_error']);
+        foreach (['debug', 'default_ttl', 'private_headers', 'allow_reload', 'allow_revalidate', 'stale_while_revalidate', 'stale_if_error'] as $key) {
```

Comment:
No need for `foreach`, Just add all variables in same `unset()`;

https://3v4l.org/pdj6L

Reply:
or keep as is as it's more readable

Reply:
True. fixed.